[eigen_I] update exercise 1 and answer

longye-tian opened the pull request:
Dear John @jstac ,

This pull request is to update the exercise 1 and its answer in eigen_I.md according to #544 .

Best ❤️ 
Longye

longye-tian committed:
[eigen_I] update exercise 1 and answer

Dear John @jstac ,

This pull request is to update the exercise 1 and its answer in eigen_I.md according to #544 .

Best

longye-tian committed:
Merge branch 'main' into eigen_I_ex

mmcky commented:
thanks @longye-tian. 

I see that the plots and diagrams are getting removed. Could this be useful as a second part of the exercise? or have they been removed to enhance clarity?

longye-tian commented:
> thanks @longye-tian.
> 
> I see that the plots and diagrams are getting removed. Could this be useful as a second part of the exercise? or have they been removed to enhance clarity?

Hi Matt,

Yes, from the original question, there is no question related to this diagram, so I removed them for clarity. 

As you mentioned, we can also change the original question by adding some questions related to the diagrams I removed. Then, we can add this extra analysis back to the answers. 

Best,
Longye

mmcky commented:
@jstac this PR needs an editorial review to see if we want to remove the figures. When you have a chance would you mind to review?

mmcky committed:
Merge branch 'main' into eigen_I_ex

mmcky committed:
Merge branch 'main' into eigen_I_ex

jstac commented:
**This branch has merge conflicts with main and needs to be rebased.**

jstac commented:
Closing as stale. Issue #544 has since been addressed in the lecture through a different approach (verified earlier today — the exercise now clearly describes the power iteration method with a specific matrix). Thanks for the contribution, @longye-tian.
